guardian/test/slack/model/slack_message_test.dart

```diff
@@ -48,20 +48,20 @@ void main() {
       },
     );
 
-    test('fromJson() should return null if decoded map is null', () {
+    test("fromJson() returns null if decoded map is null", () {
       final result = SlackMessage.fromJson(null);
 
       expect(result, isNull);
     });
 
-    test('fromJson() should convert map to Slack message', () {
+    test("fromJson() converts a map to Slack message", () {
       final result = SlackMessage.fromJson(slackMessageMap);
 
       expect(result, equals(slackMessage));
     });
 
     test(
-      'validate() should result with false if both text and blocks are missing',
+      "validate() results with false if both text and blocks are missing",
```

Comment:
```suggestion
      "validate() returns false if both text and blocks are missing",
```
The same for all similar test descriptions.